quaint/quaint-test-macros/src/test_each_connector.rs

```diff
@@ -77,6 +77,7 @@ impl darling::FromMeta for TagsWrapper {
     }
 }
 
+#[allow(clippy::needless_borrow)]
```

Comment:
Why are we needlessly borrowing `attributes_meta`? (I assume `clippy` wasn't properly configured for `quaint`, before).

Reply:
No, it's a false positive. You cannot remove the clippy linter, it actually needs the borrow. There are some false positives from this clippy check identified as issues in the repository.